proto/lib/protocol/bep.proto

```diff
@@ -147,6 +148,29 @@ message Counter {
     uint64 value = 2;
 }
 
+message PlatformData {
+    UnixData    unix    = 1 [(gogoproto.nullable) = true];
```

Comment:
These are set as explicitly nullable because we need to differentiate between (for example) not having Unix ownership information at all and having ownership info with UID/GID zero/zero.